pkg/controller/gcp/cache/cloudmemorystore_instance.go

```diff
@@ -44,16 +44,10 @@ const (
 	controllerName = "cloudmemorystoreinstances.cache.gcp.crossplane.io"
```

Comment:
Yeah, my goal is to avoid touching actual logic in the concrete controllers wherever possible; just updating them to use the new shared embedded structs, conditioned status, etc.